File: tareas/2/24190313/SESION_6/teorema_de_pitagoras.py
import math
def hipotenusa(a,b):
    c = float(math.sqrt((a*a)+(b*b)))
    return c
a = float(input("Ingrese el valor del primer cateto: "))
b = float(input("Ingrese el valor del segundo cateto: "))
print (f"Por el teorema de Pitágoras, la hipotenusa mide: {hipotenusa(a,b)}")

File: ejercicios/01_02_03_04_05_06_07_08/teorema_de_pitagoras/teorema_de_pitagoras.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(void) {
    double a, b;
    cout << "Ingrese los valores de los catetos: ";
    cin >> a >> b;
    double c = sqrt(a * a + b * b);
    cout << "Por el teorema de Pitagoras, la hipotenusa mide " << c << endl;
    return 0;
}